frontend/public/components/_nav.scss

```diff
@@ -107,16 +129,24 @@ $active-border: 3px;
       display: inline-block;
     }
   }
+  .fa-angle-right {
```

Comment:
Nit:  alphabetization of rules here.  I recommend `&:hover, .fa-angle-right, span` be the order.